mapsforge-core/src/main/java/org/mapsforge/core/util/Parameters.java

```diff
@@ -111,6 +107,19 @@ public enum SymbolScaling {ALL, POI}
         POLYGON_EXCEPTIONS.add(new Tag("freizeitkarte", "land"));
     }
 
+    public static final class Constants {
+
+        public static final int HILLSHADING_MAGNITUDE_DEFAULT = 128;
+        public static final int HILLSHADING_COLOR_DEFAULT = 0xff000000;
```

Comment:
@Sublimis Also the hillshading constants belong in the 1 class that uses them.
They are not used anywhere else (excluding possible tests).